CHANGELOG.md

```diff
@@ -9,7 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Support for prepared statement syntaxes specific to mysql dialect [#1147](https://github.com/sqlfluff/sqlfluff/issues/1147)
+- Support for QUALIFY syntax specific to teradata dialect [#1184](https://github.com/sqlfluff/sqlfluff/pull/1184)
+  <<<<<<< HEAD
```

Comment:
This looks like an incorrectly resolved merge conflict.

Reply:
@barrywhart this has been fixed